hs_core/models.py

```diff
@@ -1510,6 +1531,30 @@ def delete_all_elements(self):
         self.sources.all().delete()
         self.relations.all().delete()
 
+    def copy_all_elements_to(self, tgt_res):
```

Comment:
@ironfroggy good point. I have changed copy_all_elements_to() to copy_all_elements_from() to modify its own metadata elements. While doing so, I have also moved one import to the top as you commented, and the other import that caused circular import is not needed any more since I have moved that metadata element update out of this function to be more logical. I will need to add unit tests and address one more issue that somehow only occurs on beta vm deployment but does not happen in my local dev env, but I think I have addressed all your comments so far. Let me know if you see any more issues. Thanks.